packages/feedback-service/src/resolver.ts

```diff
@@ -25,32 +24,86 @@ export const FeedbackResolver = {
             headers: { 'Content-Type': 'application/json' },
             body: graphql
           };
-          fetch('http://localhost:8080/graphql', requestOptions)
+          return fetch('http://localhost:8080/graphql', requestOptions)
             .then((result: any) => result.json())
             .then((output: any) => {
               const usersdata = output.data;
-              newresponse = response.map(function (res) {
-                const createdby = usersdata['rhatUUID_' + res['createdBy']];
-                const updatedby = usersdata['rhatUUID_' + res['updatedBy']];
-                res['createdBy'] = createdby;
-                res['updatedBy'] = updatedby;
+              const newresponse = response.map(function (data: any) {
+                const createdby = usersdata['rhatUUID_' + data['createdBy']][0];
+                const updatedby = usersdata['rhatUUID_' + data['updatedBy']];
+                const res = { ...{ ...data }._doc, createdBy: createdby, updatedBy: updatedby };
```

Comment:
Can return the object directly. No need to store it in a variable if there're no operations to perform.

```suggestion
                return { ...{ ...data }._doc, createdBy: createdby, updatedBy: updatedby };
```